.github/dependabot.yml

```diff
@@ -24,3 +24,93 @@ updates:
     directory: "/docs/_website/generator/"
     schedule:
       interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/examples/amiibo/frontend/workflows/amiibo/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/packages/app/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/packages/core/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/packages/data-layout/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/packages/tools/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/packages/wizard/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/workflows/ec2/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/workflows/envoy/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/workflows/experimentation/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/workflows/k8s/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/workflows/kinesis/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/frontend/workflows/sourcecontrol/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gomod"
+    directory: "/tools/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/tools/scaffolding/templates/frontend/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gomod"
+    directory: "/tools/scaffolding/templates/gateway/backend/"
+    schedule:
+      interval: "daily"
```

Comment:
the go.mod is not technically valid (syntax errors) due to the templating, i wonder how dependabot will handle that. probably same for other scaffolding package files.

Reply:
Yeah, I want to see how forgiving dependabot's parser can be.